packages/web-twig/src/Resources/components/Alert/stories/AlertCentered.twig

```diff
@@ -1,17 +1,17 @@
-<Alert color="success" isCentered class="mb-600">
+<Alert color="success" isCentered UNSAFE_className="mb-600">
```

Comment:
@crishpeen @literat I think we soon need to find a way how to apply utility classes because there is nothing unsafe about adding a margin. As stated in the docs ⬆️ , `UNSAFE` props should be reserved for edge cases, not for everyday use. I'm filing it for our weekly discussion.